pkg/controller/storagecluster/storagecluster.go

```diff
@@ -255,8 +253,13 @@ func (c *Controller) Reconcile(_ context.Context, request reconcile.Request) (re
 	}
 
 	if err := c.syncStorageCluster(cluster); err != nil {
+		// Ignore object revision conflict errors
```

Comment:
I reviewed all places where we update stc, because next reconcile loop should recover, it's better to hide this "object has been modified" error in k8s events, or show it anyway but without stc status changed to Degraded. right now I see this happens when do stc update followed by a status update e.g. https://github.com/libopenstorage/operator/blob/master/pkg/controller/storagecluster/storagecluster.go#L461
race condition could happen when updating status but revision got updated during this.

Reply:
Ok, it should not show degraded